platform/install/helm.mdx

```diff
@@ -4,53 +4,103 @@ sidebar_label: Using Helm
 sidebar_position: 2
 ---
 
-vCluster Platform can be installed directly via Helm, in fact, even when following the recommend installation
-procedure in the Getting Started Guide the vCluster CLI is actually using Helm
-to install vCluster Platform.
-
-Managing the vCluster Platform installation with Helm directly can be a great way to "GitOps" your vCluster Platform
-installation, by using ArgoCD or other GitOps tools to manage the vCluster Platform deployment via Helm and
-appropriate Helm values. This section outlines the basics of installing and managing vCluster Platform with
-Helm, and should serve as a solid starting point for managing vCluster Platform
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import LoginToPlatformAfterHelmInstall from  '../_partials/install/login-after-helm-install.mdx'
+
+The platform can be installed directly using Helm commands. Managing the platform installation with Helm directly can be a great way to "GitOps" your platform
+installation, by using ArgoCD or other GitOps tools to manage the platform deployment. This section outlines the basics of installing and managing the platform with
+Helm, and can serve as a solid starting point for managing the platform
 in a GitOps fashion as well.
 
-## Simple Installation
+## Installation
+
+### Prerequisites
+
+- [Helm v3.10+](https://helm.sh/docs/helm/helm_install/)
```

Comment:
I'm curious why we are recommending minor version as well and not just `v3`?

Reply:
Honestly, I'm not sure. It's a requirement that was listed on vcluster docs, but never asked. 

@FabianKramm Do you have any ideas on why the minor version would matter? 

@Piotr1215 I don't have a preference either way, but would like the pre-reqs between vcluster and vcluster platform docs to match.

Reply:
I think we used capabilities for a while which was I believe a feature they added later on, but since we removed that I guess its fine to just say v3+